atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/thrift/Mutations.java

```diff
@@ -25,7 +25,7 @@ private Mutations() { }
 
     public static Mutation rangeTombstoneForColumn(byte[] columnName, long maxTimestampExclusive) {
         Deletion deletion = new Deletion()
-                .setTimestamp(Long.MAX_VALUE)
+                .setTimestamp(maxTimestampExclusive)
```

Comment:
Should this be + 1?

Reply:
This is an exclusive bound, so it's the timestamp of the write that triggered the targeted sweep, so all swept values will be at a timestamp less than this one. Effectively, maxTimestampExclusive is 1 + the maximum timestamp of any cell that could be swept in this request.

Reply:
Makes sense